ConsistencyManager/ConsistencyManager.swift

```diff
@@ -388,7 +388,7 @@ public class ConsistencyManager {
     */
     public func cleanMemory() {
         dispatchTask { _ in
-            NSNotificationCenter.defaultCenter().postNotificationName(ConsistencyManager.kCleanMemoryAsynchronousWorkStarted, object: self)
+            NotificationCenter.default.post(name: Notification.Name(rawValue: ConsistencyManager.kCleanMemoryAsynchronousWorkStarted), object: self)
```

Comment:
Is this the right way to do it? Shouldn't we instead create a new Notification.Name? Guessing though...not familiar with swift 3 yet.